### PR TITLE
set setup.py for package data

### PR DESCRIPTION
**Pull request by tczorro**
1. set tmp version number to 0.1
2. add pkg_resource for finding test files

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def get_readme():
 
 setup(
     name='iodata',
-    version=get_version(),
+    version='0.1',
```
Comment (tovrstra) [inline setup.py]: See #11 for how to get `get_version` to work.

Comment (tczorro) [reply setup.py]: Update the commit. Change the setting back

**Comment by matt-chan** [comment]
@tczorro, is this PR ready to merge? In either case, could you rebase it on master please? We've fixed up a lot of things.

**Comment by tovrstra** [comment]
@matt-chan Definitely not ready to merge. Can you explain him how the new `setup.py` works?

**Comment by matt-chan** [comment]
Will do. I think he arrives in Canada today.

**Comment by tovrstra** [comment]
Great! thanks.

**Pull request closed without merging**